index.js

```diff
@@ -55,7 +56,9 @@ module.exports = (input, output, opts) => {
 	opts = Object.assign({plugins: []}, opts);
 	opts.plugins = opts.use || opts.plugins;
 
-	return globby(input, {nodir: true}).then(paths => Promise.all(paths.map(x => handleFile(x, output, opts))));
+	return globby(input, {nodir: true})
+		.then(paths => Promise.resolve(paths.filter(path => junk.not(path))))
```

Comment:
The `Promise.resolve` call is moot and you can just put the filter logic in the same `.then` as the `Promise.all` call.